src/client/graphql/PotentialCrossSells.graphql

```diff
@@ -3,8 +3,8 @@ query PotentialCrossSells($locale: Locale!) {
     potentialCrossSells(locale: $locale) {
```

Comment:
Nothing to consider now but another way to accomplish this is to add a filter to this query like `potentialCrossSells(locale: $locale, filter: WEB)` -- what do you think about that?

Reply:
Yeah, and then in Giraffe swap out the fields from PCMS? I like that solution more 👍 
Especially since we also might have to have different cross-sells in the different platforms..

Reply:
Yeah exactly, seem slightly more scalable -- but we can start with this if you want!

Reply:
yeah, let's start here. And I'll probably introduce `filter` for when we need web-specific promos altogether.